spec/facades/attendance_show_facade_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe AttendanceShowFacade do
+  before :each do
+    @turing_module = create(:setup_module)
+    @attendance = create(:attendance, turing_module: @turing_module)
+    @facade = AttendanceShowFacade.new(@attendance)
+  end
+  
+  describe '#turing_module_id'do
+    it 'returns the turing module id' do
+      expect(@facade.turing_module_id).to eq(@turing_module.id)
+    end
+  end
+
+  describe '#unasigned_zoom_aliases' do
+    it 'returns an array of unclaimed zoom aliases' do
```

Comment:
Can we make this "it" statement more descriptive of the test. I.e, provide an explanation that only zoom_aliases from the corresponding attendance will be included.

Reply:
Great call. I will update the description!